src/data/AmendmentCenter.hpp

```diff
@@ -132,6 +132,8 @@ struct Amendments {
     REGISTER(fixAMMv1_2);
     REGISTER(AMMClawback);
     REGISTER(Credentials);
+    REGISTER(DynamicNFT);
+    REGISTER(PermissionedDomains);
```

Comment:
I'm not sure whether this is correct, because you don't add support for this amendment to Clio. Maybe we should temporary allow amendment center test to fail instead.

Reply:
Yes. Here is weird.

Reply:
Added TODO for now.